ibis/pandas/tests/test_operations.py

```diff
@@ -50,6 +50,32 @@ def test_join(how, left, right, df1, df2):
     tm.assert_frame_equal(result[expected.columns], expected)
 
 
+def test_asof_join(time_left, time_right, time_df1, time_df2):
```

Comment:
I would just skip these tests on the versions of pandas we test that don't have `merge_asof`.

Reply:
something like this?

    merge_asof_minversion = pytest.mark.skipif(
        pd.__version__ < '0.19.2',
        reason="at least pandas-0.19.2 required for merge_asof")

    @merge_asof_minversion
    def test_asof_join...